src/api/spec/factories/packages.rb

```diff
@@ -114,18 +115,20 @@
 
     factory :package_with_kiwi_file do
       transient do
-        kiwi_file_content '<?xml version="1.0" encoding="UTF-8"?>
-<image name="Christians_openSUSE_13.2_JeOS" displayname="Christians_openSUSE_13.2_JeOS" schemaversion="5.2">
-  <description type="system">
-    <author>Christian Bruckmayer</author>
-    <contact>noemail@example.com</contact>
-    <specification>Tiny, minimalistic appliances</specification>
-  </description>
-  <preferences>
-    <type image="docker" boot="grub"/>
-    <version>2.0.0</version>
-  </preferences>
-</image>'
+        kiwi_file_content do
+          '<?xml version="1.0" encoding="UTF-8"?>
+        <image name="Christians_openSUSE_13.2_JeOS" displayname="Christians_openSUSE_13.2_JeOS" schemaversion="5.2">
+          <description type="system">
+            <author>Christian Bruckmayer</author>
+            <contact>noemail@example.com</contact>
+            <specification>Tiny, minimalistic appliances</specification>
+          </description>
+          <preferences>
+            <type image="docker" boot="grub"/>
+            <version>2.0.0</version>
+          </preferences>
+        </image>'
+        end
```

Comment:
Nitpick: Also here. Changes of indentation not needed and not related to the pull request. But I don't care if the don't change the result of the tests.